api/managementpb/dbaas/xtradb_cluster.proto

```diff
@@ -129,6 +121,43 @@ message CreateXtraDBClusterRequest {
       string_not_empty: true
     }
   ];
+}
+
+message ShowXtraDBClusterResponse {
+  // Cluster name.
+  string name = 1;
+  // Cluster state.
+  XtraDBClusterState state = 2;
+  // Currently running operation, if any.
+  RunningOperation operation = 3;
+  // Cluster parameters.
+  XtraDBClusterParams params = 4;
+  // ProxySQL username.
+  string username = 5;
+  // ProxySQL password.
+  string password = 6;
+  // ProxySQL host.
+  string host = 7;
+  // ProxySQL port.
+  int32 port = 8;
```

Comment:
Those 4 parameters should be a separate message type as we will likely return several sets of credentials

Reply:
what do you mean by several sets of credentials?

Reply:
Several username/password pairs for different access rights permissions. Several host/port pairs for different access methods (direct connections vs going through the proxy). Etc.

We shouldn't make this field a repeated field now, but let's extract a separate message type